.idea/runConfigurations/Unittests__Full.xml

```diff
@@ -7,14 +7,14 @@
       <env name="PYTHONUNBUFFERED" value="1" />
       <env name="UNITTEST_FLAG" value="1" />
     </envs>
-    <option name="SDK_HOME" value="$PROJECT_DIR$/venv37/bin/python" />
+    <option name="SDK_HOME" value="" />
     <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
-    <option name="IS_MODULE_SDK" value="false" />
+    <option name="IS_MODULE_SDK" value="true" />
     <option name="ADD_CONTENT_ROOTS" value="true" />
     <option name="ADD_SOURCE_ROOTS" value="true" />
     <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
     <option name="SCRIPT_NAME" value="unittest" />
-    <option name="PARAMETERS" value="discover -v -s tests" />
+    <option name="PARAMETERS" value="discover -v -s tests -p &quot;*test_questionnaire_response_dao*&quot;" />
     <option name="SHOW_COMMAND_LINE" value="false" />
     <option name="EMULATE_TERMINAL" value="false" />
     <option name="MODULE_MODE" value="true" />
```

Comment:
Looks like changes to the full unit test configuration got uploaded.

Reply:
Ah, it's submitted by accident...